app/controllers/spree/admin/variants_controller.rb

```diff
@@ -32,6 +32,7 @@ def update
 
       def new
         @url_filters = ::ProductFilters.new.extract(request.query_parameters)
+        @object.shipping_category = DefaultShippingCategory.find_or_create
```

Comment:
Please correct me if I'm wrong.
This will be used to select a default Shipping category when the user wants to create a new variant, right?

What if the master variant refers to a shipping category other than the default one? will the new variant refer to the default shipping category?

Reply:
Master variants are gone :smile: 

This is on the page where the new variant form is shown. Setting this here ensures there won't be a `nil` value for it, and sets the dropdown on the page but a different one can be selected before the form is submitted.

Reply:
I was thinking it might be helpful to default to a sibling variant's shipping method. But we plan on changing the data model further again so there's not much point doing that now.